flink-autoscaler/src/main/java/org/apache/flink/autoscaler/tuning/MemoryTuning.java

```diff
@@ -243,28 +244,36 @@ private static MemorySize adjustNetworkMemory(
             Configuration config,
             MemoryBudget memBudget) {
 
-        final long buffersPerChannel =
+        final int buffersPerChannel =
                 config.get(NettyShuffleEnvironmentOptions.NETWORK_BUFFERS_PER_CHANNEL);
-        final long floatingBuffers =
+        final int floatingBuffers =
```

Comment:
The reason this was long is that it's going to be converted to long every time when multiplying with the memory segment size.

Reply:
Currently, all types in `calculateNetworkSegmentNumber` is int, including: `currentVertexParallelism`, `otherVertexParallelism`, `buffersPerChannel` and `floatingBuffers`.

If we change `buffersPerChannel` and `floatingBuffers` to  long, and don't change `currentVertexParallelism`, `otherVertexParallelism`. It still needs to convert int to long.

How about keeping it as is? Inside of calculateNetworkSegmentNumber is int, we only need to convert int to long when ` * memorySegmentBytes`.

Reply:
No objections.